indico/modules/receipts/client/js/printing/PrintReceiptsModal.jsx

```diff
@@ -98,6 +116,10 @@ export default function PrintReceiptsModal({onClose, registrationIds, eventId})
         : {}
     );
     form.change('filename', defaultFilename || formatters.slugify(title));
+    // preload event images if necessary
+    if (!eventImages.length && customFields.some(f => f.type === 'image')) {
+      fetchImages();
+    }
```

Comment:
Check the `useIndicoAxios` docs: You can pass `{manual: !customFields.some(f => f.type === 'image')}` as the second argument to prevent the request from being executed automatically if there are no images